### DISCUSSION TITLE
Main Option창

### DISCUSSION THREAD

**Discussion by seong0929** [SeoulTechTCPGame/HumanSurvival#206] [Q&A]
- Joystick Display, Show Damage, Hide Stage Progress, Visual Effects(VFX), Performance, Classic Sound Effect의 기능이 무엇이며 현재 우리 프로젝트에서 필요한 기능인가요?
  - ![image](https://user-images.githubusercontent.com/50050003/232689501-c9dcc109-e6bd-45ca-9c78-ffc37a5d2bb6.png)
  - ![image](https://user-images.githubusercontent.com/50050003/232689478-f419f5ee-0715-4f67-a595-50190e15f21d.png)
- 사운드와 뮤직 에셋이 있나요?

**Comment by pianoop** [comment]
![image](https://user-images.githubusercontent.com/86285421/233768039-39cef973-dc98-49b9-9a43-a1dd6c23478c.png)  
Load last save의 정확한 기능이 무엇인가요?
